CMakeLists.txt

```diff
@@ -93,8 +93,8 @@ if(GTEST)
     enable_testing()
 endif()
 
-set(CMAKE_BUILD_RPATH /usr/local/lib/gcc12)
-set(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++")
+# set(CMAKE_BUILD_RPATH /usr/local/lib/gcc12)
+# set(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++")
```

Comment:
ici, je ne comprends pas pourquoi c'était présent? utiliser gcc12 le fait par défaut non? ça nous bloque gcc13 en plus si je ne me trompe pas?

Et pourquoi link de manière statique avec la librairie standard? (c'est risqué de faire ça avec glibc :/)

Reply:
I think this is an old artefact of a FreeBSD fix.

Reply:
ok ok, hésite pas a pull de ton coté et aa me dire si ça casse tout quand j'aurais ouvert la pr